README.md

```diff
@@ -694,21 +694,21 @@ someData
   .dispose(in: bag)
 ```
 
-We would like to do the loading on another queue. We could dispatch async the loading, but what if we cannot change the signal producer closure because it is defined in a framework or there is another reason we cannot change it. That is when the operator `executeOn` saves the day.
+We would like to do the loading on another queue. We could dispatch async the loading, but what if we cannot change the signal producer closure because it is defined in a framework or there is another reason we cannot change it. That is when the operator `subscribe(on:)` saves the day.
 
 ```swift
 someData
-  .executeOn(.global(qos: .background))
-  .observeOn(.main)
+  .subscribe(on: ExecutionContext.global(qos: .background))
+  .receive(on: ExecutionContext.main)
   .observeNext { data in // does not block current thread
     display(data)
   }
   .dispose(in: bag)
 ```
 
-By applying `executeOn` we define where the signal producer gets executed. We usually use it in a combination with `observeOn` to define where the observer receives events.
+By applying `subscribe(on:)` we define where the signal producer gets executed. We usually use it in a combination with `receive(on:)` to define where the observer receives events.
 
-Note that there are also operators `observeIn` and `executeIn`. Those operators are similar to the ones we described with the difference that they work with execution contexts instead of with dispatch queues. Execution context is a simple abstraction over a thread or a queue. You can see how it is implemented [here](https://github.com/DeclarativeHub/ReactiveKit/blob/master/Sources/ExecutionContext.swift).
+Note that these operators work with execution contexts. Execution context is a simple abstraction over a thread or a queue. You can see how it is implemented [here](https://github.com/DeclarativeHub/ReactiveKit/blob/master/Sources/ExecutionContext.swift).
```

Comment:
I didn't know how to appropriately paraphrase this, but I thought it would be useful to keep the link to ExecutionContext.